index.html

```diff
@@ -7,12 +7,59 @@
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="style.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+  <nav class="navigation"> 
+    <div>
+      <ul class= "links">
+        <li class="current-page"><a href="">Meet Karma</a></li>
```

Comment:
what does the "current-page" class refer to? There doesn't seem to be an associated entry in your css file. Instead of current-page, look at using pseudo classes to capure hovering. 

Consider as well as looking at applying a class to all the li elements and looking at how you would remove the text decoration to make the menu options look less like links and more like menu items.